packages/amplify-cli/src/commands/build-overrides.ts

```diff
@@ -1,27 +1,47 @@
+import { $TSContext, overriddenCategories, IAmplifyResource } from 'amplify-cli-core';
+import { printer } from 'amplify-prompts';
+import { transformCfnWithOverrides } from 'amplify-provider-awscloudformation';
+
 /**
  * Command to transform CFN with overrides
  */
 const subcommand = 'build-overrides';
 
-module.exports = {
-  name: subcommand,
-  run: async context => {
-    try {
-      const {
-        parameters: { options },
-      } = context;
-      await context.amplify.executeProviderUtils(context, 'awscloudformation', 'buildOverrides', {
-        forceCompile: true,
-      });
-    } catch (error) {
-      context.print.error(error.message);
+export const run = async (context: $TSContext) => {
+  try {
+    await transformCfnWithOverrides(context, 'auth', 'extauth38706339487063394');
+  } catch (err) {
+    printer.error(err.stack);
+    printer.error('There was an error overriding the resources');
+    context.usageData.emitError(err);
+    process.exitCode = 1;
+  }
 
-      if (error.stack) {
-        context.print.info(error.stack);
-      }
+  // const resourceName = context?.input?.subCommands?.[0];
```

Comment:
leftover?

Reply:
I have done this for testing , will uncomment below code after